Resources/Prototypes/Entities/Clothing/Multiple/towel.yml

```diff
@@ -0,0 +1,764 @@
+- type: entity
+  id: BaseTowel
+  name: base towel
+  abstract: true
+  description: If you want to survive out here, you gotta know where your towel is.
+  parent: [ UnsensoredClothingUniformBase, ClothingHeadBase, ClothingBeltBase ]
+  components:
+  - type: Sprite
+    sprite: Clothing/Multiple/towel.rsi
+  - type: Clothing
+    sprite: Clothing/Multiple/towel.rsi
+    slots: 
+    - BELT
+    - INNERCLOTHING
+    - HEAD
+    femaleMask: UniformTop
+    equipSound: 
+    unequipSound: 
+  - type: Spillable
+    solution: absorbed
+  - type: Absorbent
+    pickupAmount: 15
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 30
+        reagents:
+        - ReagentId: Fiber
+          Quantity: 30
+      absorbed:
+        maxVol: 30
+  - type: Fiber
+    fiberColor: fibers-white
+  - type: DnaSubstanceTrace
+  - type: Item
+    size: Small
+
+- type: entity
+  id: TowelColorWhite
+  name: white towel
+  parent: BaseTowel
+  components:
+  - type: Sprite
+    layers:
+    - state: icon
+      color: "#EAE8E8"
+  - type: Item
+    inhandVisuals:
+      left:
+      - state: inhand-right
```

Comment:
This should be 'inhand-left' for the left hand. This needs to be adjusted for the rest of the entries

Reply:
Huh, wonder how that happened.